pkg/schedule/schedulers/hot_region_v2.go

```diff
@@ -141,7 +181,7 @@ func (bs *balanceSolver) setSearchRevertRegionsV2() {
 	}
 }
 
-// calcProgressiveRank calculates `bs.cur.progressiveRank`.
+// calcProgressiveRankV2 calculates `bs.cur.progressiveRank`.
 // See the comments of `solution.progressiveRank` for more about progressive rank.
```

Comment:
Where is `solution.progressiveRank`?
And can you change below word `idWorsened` to comparative rather than adjective, in this way we can unify `isBetter` with `isWorsen`

Reply:
Oh, my poor vocabulary made confused :)